routes/api.php

```diff
@@ -12,10 +12,15 @@
 | is assigned the "api" middleware group. Enjoy building your API!
 |
 */
+Route::group(['as' => 'api.'], function (){
+    Route::post('register', 'AuthController@register')->name('register');
+    Route::post('login', 'AuthController@login')->name('login');
+    Route::post('logout', 'AuthController@logout')->name('logout')->middleware('auth:sanctum');
+    Route::get('user', 'AuthController@user')->name('getUser')->middleware('auth:sanctum');
 
-Route::post('register', 'AuthController@register')->name('register');
-Route::post('login', 'AuthController@login')->name('login');
-Route::post('logout', 'AuthController@logout')->name('logout')->middleware('auth:sanctum');
-Route::get('user', 'AuthController@user')->name('getUser')->middleware('auth:sanctum');
+    Route::post('/offers', "OfferController@store");
+});
+
```

Comment:
Please, avoid making changes directly to other people's code unless it's a bug and use the review comments to communicate your requested changes.
This change would not be necessary if you just moved the new routes to web routes, but hopefully now we won't need to after the changes.

Reply:
I have checked that it will not effect any thing I just removed it from ApiServiceProvider and added it here so it won't be global.